tests/files/mpeg.rs

```diff
@@ -306,3 +307,57 @@ fn remove_id3v1() {
 fn remove_ape() {
 	crate::remove_tag!("tests/files/assets/minimal/full_test.mp3", TagType::Ape);
 }
+
+#[test]
+fn read_and_write_tpil_frame() {
+	let key_value_pairs = vec![
+					("engineer".to_string(), "testperson".to_string()), 
+					("vocalist".to_string(), "testhuman".to_string())
+				];
+
+	let mut file = temp_file!("tests/files/assets/issue_213.mp3");
+
+	let tagged_file = Probe::new(&mut file)
+		.options(ParseOptions::new().read_properties(false))
+		.guess_file_type()
+		.unwrap()
+		.read()
+		.unwrap();
+
+	assert_eq!(tagged_file.file_type(), FileType::Mpeg);
+
+	let mut tag: Id3v2Tag = tagged_file.tag(TagType::Id3v2).unwrap().clone().into();
+	
+	tag.insert(
+		Frame::new(
+			"TIPL", 
+			KeyValueFrame {
+				encoding: lofty::TextEncoding::UTF8,
+				key_value_pairs: key_value_pairs.clone()
+			},
+	 		FrameFlags::default()
+		).unwrap()
+	);
+
+	file.rewind().unwrap();
+	tag.save_to(&mut file).unwrap();
+
+	// Now reread the file
+	file.rewind().unwrap();
+	let tagged_file = Probe::new(&mut file)
+		.options(ParseOptions::new().read_properties(false))
+		.guess_file_type()
+		.unwrap()
+		.read()
+		.unwrap();
+	
+
+	let tag: Id3v2Tag = tagged_file.tag(TagType::Id3v2).unwrap().clone().into();
```

Comment:
The TIPL frame won't exist since you're reading a `TaggedFile`. The conversion from `Id3v2Tag` -> `Tag` explicitly discards it here:
```rs
						FrameValue::KeyValue(_) | FrameValue::UniqueFileIdentifier(_) => {
							return true; // Keep unsupported frame
						},
```
You'd have to read an `MpegFile` to use the unaltered `Id3v2Tag` directly.

It may (sometime in the future) be worth taking values from the TIPL to convert to generic `ItemKey`s, such as `TIPL:arranger` mapping to `ItemKey::Arranger`, etc.

Reply:
The test should verify that the incomplete/asymmetric implementation should not do any harm to existing files for the read-modify-write use case.